_oasis

```diff
@@ -22,6 +25,7 @@ Library "cephes"
   BuildTools:       cephes_bindgen
 
 Executable "cephes_bindgen"
+  Build$:           flag(tests)
```

Comment:
I can understand why the above are relevant, but how does this influence the opam package generation? Sorry, I've just never used `oasis2opam` before and don't understand the details.